plugins/modules/repository.py

```diff
@@ -112,6 +112,12 @@
     - Repository SSL client private key
     required: false
     type: str
+  download_concurrency:
+    description:
+      - download concurrency for sync from upstream
+      - as the API does not return this value, this will break idempotency for this module
```

Comment:
```suggestion
      - as the API does not return this value, this will break idempotence for this module
```
I guess, we don't have a version, where we expect this to be "fixed" in Katello?

Reply:
Not yet, I just asked it on the foreman support matrix, as I wasn't sure whether this API is provided by Katello/Foreman or Pulp :sweat_smile: 

Once I know that, I can file the enhancement :-)

Reply:
These modules never talk to Pulp directly, so Katello needs to handle it (and hand it through).